src/components/UI/WorkspaceSwitcher.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div>
+    <select v-model="selectedWorkspace" @change="this.updateActiveWorkspace">
+      <option value="">Personal Workspace</option>
+      <option v-for="workspace in this.user.organizations" :key="workspace.id" :value="workspace.shortcode">{{workspace.name}}</option>
+    </select>
+    <user-properties ref="userProperties"></user-properties>
+  </div>
+</template>
+
+<script>
+import { mapState, mapActions } from "vuex";
+
+export default {
+  name: "WorkspaceSwitcher",
+  computed: {
+      ...mapState('auth', ["activeWorkspace", "user"]),
+  },
+  data() {
+    return {
+      selectedWorkspace: "",
+    }
+  },
+  methods: {
+    ...mapActions('auth', ['setActiveWorkspace']),
+    updateActiveWorkspace() {
+      this.setActiveWorkspace(this.selectedWorkspace);
+      this.$router.push({name: 'Home', params: { org: this.activeWorkspace }});
```

Comment:
The parent for this is `App.vue`. Maybe to avoid making it too big, we can have another file or maybe keep the redirect logic here only for now. What do you think?

I think once we have more pages and a better navigation in place, then we can figure it out. I was thinking more in terms of Notion that instead of dropdown, we should be showing all the organizations in a sidebar or something and user will click on those links. So they would see the organization home screen when they switch workspace.